src/main/kotlin/no/nav/familie/ef/sak/repository/TilkjentYtelseRepository.kt

```diff
@@ -45,4 +45,17 @@ interface TilkjentYtelseRepository : RepositoryInterface<TilkjentYtelse, UUID>,
             AND aty.stonad_tom >= :datoForAvstemming
     """)
     fun finnKildeBehandlingIdFraAndelTilkjentYtelse(datoForAvstemming: LocalDate, sisteBehandlinger: List<UUID>): List<Pair<Long, Long>>
+
+    // language=PostgreSQL
+    @Query("""
+        SELECT ty.*
+        FROM tilkjent_ytelse ty 
+            JOIN behandling b ON b.id = ty.behandling_id
+            JOIN fagsak f ON b.fagsak_id = f.id
+        WHERE f.stonadstype = :stønadstype  AND b.status = 'FERDIGSTILT' AND b.type IN ('FØRSTEGANGSBEHANDLING', 'REVURDERING')
+         AND EXISTS (SELECT 1 FROM andel_tilkjent_ytelse aty
+                        WHERE ty.id = aty.tilkjent_ytelse AND aty.stonad_tom >= :datoForAvstemming) 
+          """)
+    fun finnTilkjentYtelserTilKonsistensavstemming(stønadstype: Stønadstype, datoForAvstemming: LocalDate): List<TilkjentYtelse>
```

Comment:
Denne må fikses så att den kun henter tilkjent til siste behandlingen per fagsak

Reply:
Fikset i f36fac3

Reply:
@charliemidtlyng Er det riktige statuser, typer som sjekkes her? 
```
AND b.status = 'FERDIGSTILT'
AND b.type IN ('FØRSTEGANGSBEHANDLING', 'REVURDERING')
```

Reply:
Tror det, men ikke mega-trivielt å skjønne hva som skjer :D 

Hvis man har både førstegangsbehandling og deretter teknisk opphør - klarer vi da å skjønne at det ikke er en tilkjentYtelse som skal plukkes ut 🤯

Reply:
Hvis man har både førstegangsbehandling og revurdering - risikrerer vi å hente samme ytelse to ganger?

Reply:
Må nok flytte rad 61 till 62, sånn att man henter siste behandlingen for alle fagsaker, og sen filtrerer man de behandlingene man ønsker